packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -47,6 +47,12 @@ export const enableSchedulingProfiler = __VARIANT__;
 // so we don't need to use __VARIANT__ to get extra coverage.
 export const replayFailedUnitOfWorkWithInvokeGuardedCallback = __DEV__;
 
+// This flag only exists so it can be connected to a www GK that acts as a
+// killswitch. We don't run our tests against the `true` value because 1) it
+// affects too many tests 2) it shouldn't break anything. But it is mildly
+// risky, hence this extra precaution.
+export const revertRemovalOfSiblingPrerendering = false;
```

Comment:
@tyao1 we need to confirm with @acdlite whether this is actually true now. We've landed a lot of changes since this, and it may no longer be safe to not run tests for it.

Reply:
I turned the flag on and ran the tests, the failures seem to only be related to rendering siblings.